default.nix

```
@@ -10,7 +10,7 @@
   nanopos = pkgs.callPackage ./pkgs/nanopos { };
   spark-wallet = pkgs.callPackage ./pkgs/spark-wallet { };
   electrs = (pkgs.callPackage ./pkgs/electrs { }).rootCrate.build;
-  elementsd = pkgs.callPackage ./pkgs/elementsd { };
+  elementsd = pkgs.callPackage ./pkgs/elementsd { withGui = false; };
```

what do you think about having an attribute for the gui version?

```suggestion
  elementsd = pkgs.callPackage ./pkgs/elementsd { withGui = false; };
  elements = pkgs.callPackage ./pkgs/elementsd { withGui = true; };
```